EngineLayer/ClassicSearch/ClassicSearchEngine.cs

```diff
@@ -62,101 +65,245 @@ protected override MetaMorpheusEngineResults RunSpecific()
 
             if (Proteins.Any())
             {
-                int maxThreadsPerFile = CommonParameters.MaxThreadsToUsePerFile;
-                int[] threads = Enumerable.Range(0, maxThreadsPerFile).ToArray();
-                Parallel.ForEach(threads, (i) =>
+                if (WriteSpetrcalLibrary == false)
```

Comment:
There are a lot of changes in this file that are making it difficult to follow along. I'll take a look later...